src/app/pages/dynamic-stepper/dynamic-stepper.component.scss

```diff
@@ -5,39 +5,47 @@
     flex-direction: column;
 }
 
+.stepper-container {
+  ::ng-deep .mat-step-header:hover,
+  ::ng-deep .mat-focus-indicator.add-button.mat-flat-button.mat-button-base.ng-star-inserted {
+    background-color: unset;
+  }
+}
```

Comment:
Should this be in the theme?

Reply:
I think you're right, but only if I can fix the hover error from above.  Otherwise it'll be worse than before.

Reply:
These style fixes were migrated to this PR:   https://github.com/pxblue/themes/pull/103

When the other PR gets merged in, this branch will be updated to use a new version of @pxblue/angular-themes.